theme/static/css/hydroshare_core.css

```diff
@@ -1604,10 +1604,6 @@ div#sharing-status {
     overflow: hidden; /* Fixes height for floating elements */
 }
 
-.info-group {
-	width: calc(100% - 220px);
-}
-
```

Comment:
@mauriel133 By any chance, did you forget to check in some files that reference this info-group class style? For example resource-header.html refer to this info-group class to place buttons. From git display of changes, I only see this class style definition is removed, but I don't see reference to this removed style class is removed or replaced by other styles.